WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/builders/ThreatItemBuilder.kt

```diff
@@ -83,6 +91,14 @@ class ThreatItemBuilder @Inject constructor() {
             }
         }
     }
+
+    fun buildThreatItemSubHeaderColor(threatModel: ThreatModel) =
```

Comment:
Minor (🔍): Consider making this function private if you know it is not going to be used as public in the future. Else, ignore this comment.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13895/commits/dff118754c3e129bd8648712002e73a0751d8dda